src/components/DeviceChecker/DeviceChecker.vue

```diff
@@ -71,7 +71,7 @@
 							:audio-enabled="audioOn"
 							:current-volume="currentVolume"
 							:volume-threshold="volumeThreshold"
-							:overlay-color="'#999999'" />
+							overlay-muted-color="#888888" />
```

Comment:
@nickvergessen please check it:
this color overwrites default color in both light and dark themes, but looks clearly visible for me